src/main/java/com/exasol/adapter/dialects/bigquery/BigQueryQueryRewriter.java

```diff
@@ -51,26 +40,21 @@ public BigQueryQueryRewriter(final SqlDialect dialect, final RemoteMetadataReade
     public String rewrite(final SqlStatement statement, final List<DataType> selectListDataTypes,
             final ExaMetadata exaMetadata, final AdapterProperties properties) throws AdapterException, SQLException {
         final String query = getQueryFromStatement(statement, properties);
-        LOGGER.fine(() -> "Query to rewrite: " + query);
-        final StringBuilder builder = new StringBuilder();
+        LOGGER.fine(() -> "Query to rewrite: '" + query + "'");
```

Comment:
```suggestion
        LOGGER.fine(() -> String.format("Query to rewrite: '%s'", query));
```

Or maybe LOGGER.fine(() -> "Query to rewrite: '{}'", query);
Not sure how it should look lambda expression.

Reply:
I will do this in the follow-up PR, ok?